### PR TITLE
NETOBSERV-105 Netflow table sort test

### PR DESCRIPTION
**Pull request by OlivierCazade**
Added tests for netflow-table sorting and found a small bug in netflow-table-raw keys.

### CODE REVIEW

File: web/src/components/__tests__/netflow-table-header.spec.tsx [hunk 1]
```diff
@@ -40,4 +58,18 @@ describe("<NetflowTableHeader />", () => {
...
+      <NetflowTableHeaderWrapper
+        onSort={onSort}
+        sortIndex={0}
+        sortDirection={"asc"}
+        columns={ColumnsSample}
+      />
+    );
+    expect(wrapper.find(NetflowTableHeader)).toBeTruthy();
+    wrapper.find("button").at(0).simulate("click");
+    expect(onSort).toHaveBeenCalled();
```
Comment (jpinsonneau) [inline web/src/components/__tests__/netflow-table-header.spec.tsx:73]: Should we check what sort has been requested by the click ?
You can use `toHaveBeenCalledWith` to check onSort arguments

File: web/src/components/netflow-table-row.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ const NetflowTableRow: React.FC<{ flow: ParsedStream; columns: Column[] }> = ({
   return (
     <Tr>
       {columns.map((c) => (
-        <Td key="{c.id}">{content(c)}</Td>
+        <Td key={c.id}>{content(c)}</Td>
```
Comment (jpinsonneau) [inline web/src/components/netflow-table-row.tsx:83]: good catch :smile:

File: web/src/components/__tests__/netflow-table-header.spec.tsx (hunk 1 above)
Comment (OlivierCazade) [reply web/src/components/__tests__/netflow-table-header.spec.tsx]: Done, thanks !

**Review by jpinsonneau — APPROVED** [review]
LGTM thanks @OlivierCazade

**Pull request merged**